### PR TITLE
Added es.toml

### PR DESCRIPTION
**Pull request by reymon359**
## Description

I translated the en.toml to spanish and added it to the i18n folder.

## Motivation and Context

I saw this awesome theme and decided to use it on my personal blog. At first I had not intention to contribute so I made a module as sugested and started changing the main config.toml and stuff in the content. Then I realized I needed a i18n/es.toml file and decided to create it but after adding to the module I could not "push and pull" new changes from the original repo, that's why I decided to make a pull request to the original to add it. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the [Contributing Document](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md).
- [x] My code follows the [code style](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md#Style-Guide) of this project.
- [ ] My change requires a change to the [documentation](https://github.com/pacollins/hugo-future-imperfect-slim/wiki).
- [ ] I have updated the documentation, including `theme.toml`, accordingly.

### CODE REVIEW

**Comment by pacollins** [comment]
Thanks for this! Can you go ahead and add the following to this? We are still working on the exact wording though.

https://github.com/pacollins/hugo-future-imperfect-slim/pull/51/commits/602bdbc7958b93626b141d26f0b6e713303180de#diff-37fb0a88b36c36c1795ae48027e1edf3#L49-L50

**Comment by reymon359** [comment]
I am glad it helped!😄 
Sure, I will take a look to the spanish part of this https://github.com/pacollins/hugo-future-imperfect-slim/pull/52#issuecomment-521042464

**Comment by reymon359** [comment]
Done! https://github.com/pacollins/hugo-future-imperfect-slim/commit/602bdbc7958b93626b141d26f0b6e713303180de#r34684225
If you need anything else just let me know😉

**Comment by reymon359** [comment]
I added the attribution too. If you need anything else just let me know😊

File: i18n/es.toml [hunk 1]
```diff
@@ -46,3 +46,5 @@
   other = "Página siguiente"
 [previous_page]
   other = "Página anterior"
+[attribution]
+  other = "Impulsado por <a href='//gohugo.io' target='_blank' rel='noopener'>Hugo v{{ .Site.Hugo.Version }}</a>.  Tema <a href='//themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>Hugo Future Imperfect Slim</a> adaptado de <a href='//html5up.net/future-imperfect' target='_blank' rel='noopener'>Future Imperfect</a>."
```
Comment (VincentTam) [inline i18n/es.toml]: I intended to put

```toml
[attribution]
	  other = "Designed by <a href='//themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>@pacollins</a>. Ported from <a href='//html5up.net/future-imperfect' target='_blank' rel='noopener'>HTML5UP</a>.&nbsp;<i class='fab fa-creative-commons-by'></i>&nbsp;3.0"
```

Is it OK if I change it to

```toml
[attribution]
  other = "Diseñado por <a href='//themes.gohugo.io/hugo-future-imperfect-slim/' target='_blank' rel='noopener'>@pacollins</a>. Adaptado de <a href='//html5up.net/future-imperfect' target='_blank' rel='noopener'>HTML5UP</a>.&nbsp;<i class='fab fa-creative-commons-by'></i>&nbsp;3.0"
```

Comment (reymon359) [reply i18n/es.toml]: Sorry I missed it, thank you for the advice!

**Comment by VincentTam** [comment]
@reymon359 Thanks for follow-up.  I'll leave it for @pacollins to decide.

**Comment by pacollins** [comment]
@reymon359 Again, thank you SO much for this PR. It is actually really important to me that we have proper translations. I appologize for having you adjust your PR 1 last time, but I think we have determined the syntax we will use. [See this post.](https://github.com/pacollins/hugo-future-imperfect-slim/pull/51#issuecomment-521406463) Feel free to modify the wordings to optimize the _look_ of the footer (ideally three lines).

**Comment by reymon359** [comment]
I like the new syntax you guys came up with, it is much simpler👍. 
I adjusted and translated the PR to it while to optimize it to 3 lines. 
Here is how it looks on Desktop:
image
And here on Mobile (Chrome browser's Galaxy S5):
image

**Pull request merged**